app/models/application_record.rb

```diff
@@ -0,0 +1,3 @@
+class ApplicationRecord < ActiveRecord::Base
```

Comment:
Did this get automatically generated via a console command? I'm not sure what this does, or whether it makes it any different from the rest of the models which inherit from ActiveRecord::Base directly.